Library/Homebrew/test/bottle_publisher_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "bottle_publisher"
+
+describe BottlePublisher do
+  let(:tap) { CoreTap.new }
+
+  changed_formulae_names = ["#{CoreTap.instance.name}/hello.rb"]
+  bottle_publisher = described_class.new(
```

Comment:
Make this the `subject`: https://relishapp.com/rspec/rspec-core/v/2-8/docs/subject/explicit-subject